### DISCUSSION TITLE
[Enhancement] [table] Slot for expand icon in table expandable row

### DISCUSSION THREAD

**Discussion by jonelleamio** [element-plus/element-plus#14272] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

I would like to be able to change the icon expand icon.
In my use case I also need to add a text before the icon.

### Additional comments

_(empty)_

**Comment by wldrve** [comment]
I would need the same, as I would have to add a tooltip to the expand icon and currently I have no idea how to do so. :)

**Comment by coderXMin** [comment]
你好！请问你解决了这个问题吗？我也需要这个

**Comment by bbchuoo** [comment]
Hello, I also need this feature. I noticed that others have asked about it since 2022. Why hasn't the official team responded to this issue? It's very frustrating for me.

**Comment by wjp980108** [comment]
in progress：[PR](https://github.com/element-plus/element-plus/pull/20750)

**Comment by wjp980108** [comment]
Released, please update to 2.10.0
